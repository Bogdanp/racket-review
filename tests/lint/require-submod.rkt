#lang racket/base

(require koyo/database
         (submod koyo/console dev))
