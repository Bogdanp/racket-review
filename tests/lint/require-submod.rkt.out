/require-submod.rkt:4:10:warning:require "koyo/console" should come before "koyo/database"
